epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModel.java

```diff
@@ -184,6 +184,16 @@ public final int getLayout() {
     return this;
   }
 
+  public void addTo(DiffAdapter adapter) {
+    adapter.add(this);
+  }
+
+  public void addIf(boolean condition, DiffAdapter adapter) {
```

Comment:
Would be nice if instead of `boolean` parameter it was a predicate function so you can provide a lambda. Or it could be an overload as well.